.github/workflows/integration-platform-remote-fleet-tests.yml

```
@@ -0,0 +1,569 @@
+name: Generic Remote Fleet Tests
+on:
+  workflow_call:
+    inputs:
+      collect_deployable_artifacts:
+        required: false
+        type: boolean
+        default: false
+      test_infra_key:
+        required: true
+        type: string
+      robot_config:
+        required: false
+        type: string
+        default: 'basic-standalone-noetic.json'
+      slack_thread_id:
+        required: true
+        type: string
+      backend_image_name_and_tag:
+        required: false
+        type: string
+        default: ''
+      test_timeout:
+        required: false
+        type: number
+        default: 60
+      pytest_args:
+        required: true
+        type: string
+      xray_label:
+        required: false
+        type: string
+        default: 'label'
+      provision_tf_env_repo:
+        required: true
+        type: string
+      provision_tf_env_version:
+        required: true
+        type: string
+      provision_infra_repo:
+        required: true
+        type: string
+      provision_infra_version:
+        required: true
+        type: string
+      fleet_number_members:
+        required: true
+        type: string
+      debug_fleet_keep_alive:
+        required: false
+        type: boolean
+      fleet_ips:
+        required: true
+        type: string
+      proxmox_cluster_name:
+        required: true
+        type: string
+      with_fleet_tests:
+        required: false
+        type: boolean
+        default: false
+      slack_channel:
+        required: false
+        type: string
+        default: 'C02PB9A9F45' # rnd-platform
+
+    secrets:
+      auto_commit_user:
+        required: true
+      auto_commit_pwd:
+        required: true
+      registry_user:
+        required: true
+      registry_password:
+        required: true
+      gh_token:
+        required: true
+      aws_key_id:
+        required: true
+      aws_secret_key_id:
+        required: true
+      ssh_pem_fleet_aws_vm:
+          required: true
+      slack_token_id:
+        required: true
+      jira_username:
+        required: true
+      jira_password:
+        required: true
+      xray_clientid:
+        required: true
+      xray_secret:
+        required: true
+
+env:
+  CI_INTEGRATION_CONTAINER_VERSION: "registry.cloud.mov.ai/qa/ci-tools:4.0.0.31"
+  CI_INTEGRATION_SCRIPTS_VERSION: "4.0.0.32"
+  MOBTEST_VERSION: "0.0.6.1"
```

```suggestion
  MOBTEST_VERSION: "0.0.7.2"
```

why not using latest mobtest ?